src/resources/styles.ts

```diff
@@ -6,8 +6,8 @@ export const derivedStyles = {
   "state-icon-unavailable-color": "var(--disabled-text-color)",
   "sidebar-text-color": "var(--primary-text-color)",
   "sidebar-header-background-color": "var(--primary-background-color))",
-  "sidebar-header-height": "64px",
-  "sidebar-background-color": "var(--card-background-color)",
+  "sidebar-header-height": "var(--header-height)",
+  "sidebar-background-color": "var(--paper-listbox-background-color)",
```

Comment:
Restored this to what is was, --paper-listbox-background-color.... because of a lot of themes are incompatible and i didnt change the backgroundcolor of sidebar-background-color in the first place.
So think its better to continue using --paper-listbox-background-color. Or a lot of people need to restyle/change there themes :p

Reply:
We should not use `paper-listbox-background-color`. Paper vars are not supported. If you don't have `card-background-color` a lot is not going to work properly.

Reply:
Ohh ok i see... will undo it again :p